quad/jsonarr/jsonarr.go

```diff
@@ -0,0 +1,96 @@
+package jsonarr
```

Comment:
Just call this json. There's no collision within the cayley tree AFAICS and others can rename.